packages/components/package.json

```diff
@@ -30,10 +30,12 @@
     "@types/react-router-dom": "^5.1.5",
     "@types/uuid": "^3.4.5",
     "@use-it/event-listener": "^0.1.3",
+    "axios": "^0.19.2",
```

Comment:
I could use pure XHR here instead of axios but, it mades the request easier to setup / execute. Not sure what the rest of you think?